test/quantization/core/test_quantized_op.py

```diff
@@ -3691,6 +3686,38 @@ def test_embedding_byte(self, num_embeddings, embedding_dim):
         ref = torch.embedding(weights, indices, padding_idx=-1, scale_grad_by_freq=False, sparse=False)
         torch.testing.assert_close(ref, qresult, atol=0.005, rtol=1e-3)
 
+    """ Tests the correctness of the quantized 4 bit embedding lookup operator """
+    @given(num_embeddings=st.integers(10, 100),
```

Comment:
is it possible to merge this test with test_embedding_byte?
also we don't want to use hypothesis for testing, might be good to remove hypothesis here as well and change it to simple for loops. (remove @given, example: https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L1430), can be done in a separate PR.

Reply:
Do you mean removing hypothesis for both test_embedding_byte and test_embedding_4bit or just the latter? It was already there in test_embedding_byte, so I thought I was supposed to add it. 

And yeah, I can easily combine test_embedding_byte and test_embedding_4bit. Is it acceptable to rename the function to something like test_embedding?

Reply:
> Do you mean removing hypothesis for both test_embedding_byte and test_embedding_4bit or just the latter? It was already there in test_embedding_byte, so I thought I was supposed to add it.

we can combine the tests and remove hypothesis, it can be in a separate PR

Reply:
Sounds good. I'll put that in PR5